gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/DefaultResourceManagerRpc.java

```diff
@@ -82,7 +88,7 @@ public Project get(String projectId, Map<Option, ?> options) throws ResourceMana
           .execute();
     } catch (IOException ex) {
       ResourceManagerException translated = translate(ex);
-      if (translated.code() == HTTP_FORBIDDEN) {
+      if (translated.code() == HTTP_FORBIDDEN || translated.code() == HTTP_NOT_FOUND) {
```

Comment:
Yup, 403 is returned for both not found and permission denied, and that is indeed the reason why.